### PR TITLE
Removing ruleset fetch attempts

### LINKED ISSUE #85: BAT-Client is still trying to fetch rulesets
BAT-Client is still trying to fetch rulesets from
`/v1/publisher/ruleset?consequential=true` resulting in #goaway's

### PR DESCRIPTION
**Pull request by ryanml**
Fixes #85 

Removes calls to `_updateRules`, as ruleset fetch attempts would trigger during client `sync`

### CODE REVIEW

**Comment by ryanml** [comment]
@jasonrsadler wasn't certain on what was best for this, given its unknown if these endpoints may be used again. I throw in for removing them for now though ++

cc: @evq

**Comment by jasonrsadler** [comment]
We have the history to add them back. I've already PRd a few in here to remove old code.

**Comment by ryanml** [comment]
Removed 👍

File: index.js [hunk 1]
```diff
@@ -998,13 +987,6 @@ Client.prototype._currentReconcile = function (callback) {
 
         self.state.reconcileStamp = underscore.now() + (self.state.properties.days * msecs.day)
         if (self.options.verboseP) self.state.reconcileDate = new Date(self.state.reconcileStamp)
-
-        self._updateRules(function (err) {
-          if (err) self._log('_updateRules', { message: err.toString() })
-
-          self._log('_currentReconcile', { delayTime: msecs.minute })
-          callback(null, self.state, msecs.minute)
```
Comment (jasonrsadler) [inline index.js]: This callback (to ledger.onCallback) looks like it has to be preserved to get the run loop to eventually end.

File: index.js [hunk 2]
```diff
@@ -177,15 +175,6 @@ Client.prototype.sync = function (callback) {
     } catch (ex) {}
   }
 
-  if (self.state.updatesStamp < now) {
-    return self._updateRules(function (err) {
-      if (err) self._log('_updateRules', { message: err.toString() })
-
-      self._log('sync', { delayTime: msecs.minute })
-      callback(null, self.state, msecs.minute)
```
Comment (jasonrsadler) [inline index.js]: I don't think we get in here at all. Will confirm

Comment (jasonrsadler) [reply index.js]: Scratch that. We do hit that during contribution after Tx's come in (before the votes)

**Comment by jasonrsadler** [comment]
@ryanml @evq We need to ensure that this is actually causing the error that it was intended to address. My investigations is that it is not causing the hang ups the were discussed internally. If you disagree, feel free to reopen.

**Pull request closed without merging**